Envfile

```diff
@@ -36,6 +36,9 @@ variable :REDIS_SIDEKIQ_URL, :String, default: "redis://localhost:6379"
 # Elasticsearch
 variable :ELASTICSEARCH_URL, :String, default: "http://localhost:9200"
 
+# SSL config
+variable :FORCE_SSL_IN_RAILS, String, default: "not applicable in dev"
```

Comment:
shouldn't this default to false and be a `:boolean` ? See https://gitlab.com/envied/envied/tree/master#1-configure

Reply:
I believe Rails env vars can only be strings, and I didn't really want to mess with anything not super explicit in terms of truthiness here.

So if the value equals "true" then we're good otherwise anything else is false.